vtex-apps/app-store
# Wrap product description item content in a div

#### What is the purpose of this pull request?

Wrap description item content in a single div

#### What problem is this solving?

This fixes a flex-layout in which multiple children of a `<li/>` component are positioned side-by-side

#### How should this be manually tested?
1. Visit this [App Product page](https://apps.vtex.com/google-shopping/p) and check the 3rd item of the `Comments` section. There is a link side-by-side with a text block
2. Now, visit this [App Product page](https://artur--extensions.myvtex.com/google-shopping/p) and check the 3rd item of the `Comments` section. The item elements are now inline.

#### Screenshots or example usage
**Before:**
![image](https://user-images.githubusercontent.com/3827456/81349227-f7320a00-9095-11ea-9f92-a8ed3bdc8cf1.png)
![image](https://user-images.githubusercontent.com/3827456/81349242-01ec9f00-9096-11ea-8237-6140f305cf8e.png)
**After:**
![image](https://user-images.githubusercontent.com/3827456/81349273-15980580-9096-11ea-9327-3af7bc34f2b4.png)
![image](https://user-images.githubusercontent.com/3827456/81349297-1f216d80-9096-11ea-9801-2775d56fd6eb.png)


#### Types of changes
